cmd/build/v2/services.go

```diff
@@ -24,6 +25,14 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+var (
+	// ErrimageIsNotAOktetoBuildSyntax is returned when the image is not an okteto build syntax
+	ErrimageIsNotAOktetoBuildSyntax = errors.New("image is not an okteto build syntax")
+
+	// ErrOktetBuildSyntaxImageIsNotInBuildSection is returned when the image is not in the build section
+	ErrOktetBuildSyntaxImageIsNotInBuildSection = errors.New("image is not in the build section")
```

Comment:
We remark the build section, but what would happen if the image is defined in a compose file? I mean, imagine that my dev environment is deployed with a docker-compose but I have a `test` section in the manifest referring to one of the images built through the compose? Would in this case the image being built too? Or will it just happen only if they are defined in the build section?

Reply:
When we load the compose we translate compose build sections into okteto build section, so if the test is referring to a build defined in a compose manifest it will be built.